src/routes/registerOAuth.js

```diff
@@ -0,0 +1,43 @@
+var express = require("express");
+const Users = require("../models/users.js");
+var encrypt = require("../services/encryptPassword")
+const server = express();
+
+server.use(express.json());
+
+server.post('/auth/register',
+  async (req, res) =>{
+      const { displayName, username, email, password } = req.body;
+
+      var encryptedPassword;
```

Comment:
Please use let here instead of var
```suggestion
      let encryptedPassword;
```